doc/api/toolkits/mplot3d.rst

```diff
@@ -1,3 +1,33 @@
+.. _toolkit_mplot3d-index:
+.. currentmodule:: mpl_toolkits.mplot3d
+
+*******
+mplot3d
```

Comment:
This sectioning looks awkward in the module list at https://63304-1385122-gh.circle-artifacts.com/0/doc/build/html/api/index.html

You need to:
- not use two top-level sections. (The next section should be one lebvel lower)
- Format this section analogous to the other API ReST section headers.

Reply:
thanks, fixed